src/infrastructure/delivery/index.ts

```diff
@@ -0,0 +1,24 @@
+import { providerFactory } from './provider/entities';
+import { Mail } from './provider/Mail';
+import fakeRquest from '../network/fakeRquest';
```

Comment:
Why fakeRquest instead fakeRequest?

Reply:
I can't believe it. Fixed

Reply:
> I can't believe it. Fixed

What are you doing here! Go to enjoy your weekend!